### ISSUE TITLE
False-positive UselessAssignment when in a slim capture block

### ISSUE THREAD

**Issue by jhubert** [sds/slim-lint#68]
This is a simple recreation of the issue I bumped into with a `form_for` helper in rails. I'm using capture so that it's distilled down to the essence of the issue:

```slim
- url = 'https://google.com'
= capture do
  a href=url = 'Google'
```

Running slim-lint on this gives me:

```bash
test.html.slim:2 [W] RuboCop: Useless assignment to variable - `url`.
```

**Comment by seliverstov-maxim** [comment]
+

**Comment by yjukaku** [comment]
I believe this happens with any method that takes a block, not just `capture`.

**Comment by jclusso** [comment]
Any ideas on how to fix this. Happy to open a PR with some guidence. This is really annoying since literally any time a variable is used as an attribute it fails.

```slim
- test_var = "string"
div data-test=test_var
```

**Comment by jclusso** [comment]
Any idea on how to fix this. Happy to open a PR with some guidance.

**Comment by fvue** [comment]
```
- url = 'https://google1.com'
div data=url
```
The above gives warning on line 1: RuboCop: Lint/UselessAssignment: Useless assignment to variable - `url`.
Whereas this gives no errors:
```
- url = 'https://google2.com'
div = url
```
